src/components/value-list-item/value-list-item.tsx

```diff
@@ -181,6 +184,12 @@ export class ValueListItem
    */
   @Event({ cancelable: true }) calciteListItemRemove: EventEmitter<void>; // wrapped pick-list-item emits this
 
+  /**
+   * @internal
+   */
+  @Event({ cancelable: true })
```

Comment:
This event is marked as cancelable, but there doesn't appear to be any code paths that handle cancelation. Please set to `false` if this is the case.